spec/unit/unmanaged_files_inspector_spec.rb

```diff
@@ -256,7 +259,7 @@ def expect_find_commands(system,add_files)
       end
     end
 
-    def expect_inspect_unmanaged(system,add_files,extract)
+    def expect_inspect_unmanaged(system, add_files, extract, filtered_paths = nil)
```

Comment:
Assignment Branch Condition size for expect_inspect_unmanaged is too high. [73.79/15]